.run/Start IDE with Plugin.run.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Start IDE with Plugin" type="GradleRunConfiguration" factoryName="Gradle" folderName="Plugin">
+    <ExternalSystemSettings>
+      <option name="executionName" />
+      <option name="externalProjectPath" value="$PROJECT_DIR$/packages/jetbrains-plugin" />
```

Comment:
Does Jetbrains genuinely use `$foobar$` as a substitution syntax? 😨

Reply:
I didn't realise TBF, but yeah it seems they use it for templates.